website/blog/2023-07-12-release-1.2.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Release Notes - Podman Desktop 1.2
+description: Podman Desktop 1.2 has been released!
+slug: podman-desktop-release-1.2
+authors: [cdrage]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.2 Release! 🎉
+
+We're excited to announce the release of Podman Desktop version 1.2.0! This release includes many new features (Kubernetes, Compose and extension support!), bug fixes, and improvements to enhance your container management experience. Here's a summary of the key changes in this release:
+
+- **Added start/stop/delete/restart buttons for Compose**: You can now make changes to an entire Compose group of containers
+- **Kubernetes context on the status bar**: You can now pick from multiple Kubernetes contexts
```

Comment:
Nit: "you can now" was used in previous line. Plus, you could already pick from multiple contexts (via system tray). Maybe something like  "Easier to view and easily switch the Kubernetes context"?